connect_ext_ppr/webapp.py

```diff
@@ -99,6 +113,54 @@
     },
 )
 class ConnectExtensionXvsWebApplication(WebApplicationBase):
+    # example route for creation of deployment request
+    @router.post(
+        '/deployments/requests',
+        summary='Create a new deployment request',
+        response_model=DeploymentRequestSchema,
+    )
+    def add_dep_request(
+        self,
+        deployment_request: DeploymentRequestCreateSchema,
+        background_tasks: BackgroundTasks,
+        client: ConnectClient = Depends(get_installation_client),
+        db: VerboseBaseSession = Depends(get_db),
+        installation: dict = Depends(get_installation),
+        config: dict = Depends(get_config),
+        logger: Logger = Depends(get_logger),
+    ):
+        account_id = installation['owner']['id']
+        deployment = db.query(Deployment).filter_by(id=deployment_request.deployment.id).first()
```

Comment:
[COD-B] Better to use `.one_or_none()` instead of `.first()` to be able to catch duplicates in database.

P.S. Also we already have `get_deployment_by_id` function that gets deployment and raises error if it is not found. I propose to do refactoring in future (not now) and to use the same approach everywhere (this one or previous one).